src/test/java/com/academy/ui/profilePage/EditClubCardWithManagerTest.java

```diff
@@ -350,4 +351,42 @@ public void checkStepThreeEditDescriptionTextArea() {
 
         softAssert.assertAll();
     }
+
+    @Test
+    @Issue("TUA-958")
```

Comment:
```suggestion
    @Test
    @Issue("TUA-958")
    @Description("") 
```


You can add one more annotation which gives additional information about what you will be doing in this test.